scripts/daqutils

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+
+# Go to source directory to make python import works
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+cd $SCRIPT_DIR
```

Comment:
Another tiny nitpick, can ignore
```suggestion
cd -- "$( dirname -- "${BASH_SOURCE[0]}" )"
```

Reply:
This is a tricky one and sorry for my ignorance (I basically copied this from the original restartdaq script). I think it tries to use either the directory of this source file (not sure about this, but maybe users can run it from somewhere else?) or the current directory. It also hushes the output. 
Now thinking about it, I'd probably want to expand it but given that it works in the previous restartdaq, we probably can keep it this way.